examples/demo_bgp.pp

```diff
@@ -19,7 +19,29 @@
   # --------------------------------------------------------------------------#
   # Configure Global BGP                                                      #
   # --------------------------------------------------------------------------#
-  cisco_bgp { '55.77 blue':
+
+  $maxas_limit = $operatingsystem ? {
+    'nexus' => '50',
+    default => undef
```

Comment:
Question for my own curiosity - if we default to `default` instead of `undef`, does that still work with our code?